pkg/eventprocessor/impl/eventprocessor.go

```diff
@@ -296,6 +313,43 @@ func (ep *EventProcessor) executeBlock(ctx context.Context, block eventfeed.Bloc
 	return nil
 }
 
+// executeWebhook will iterate over the receipts and send a webhook for each
+// receipt. We do this in a separate goroutine to avoid blocking.
+func (ep *EventProcessor) executeWebhook(ctx context.Context, receipts []eventprocessor.Receipt) {
+	for _, r := range receipts {
+		go func(r eventprocessor.Receipt) {
+			successTmpl := "**Error processing Tableland event:**\n\n" +
```

Comment:
suggestions:
- Put this logic inside the Webhook object. Right now the Webhook is just responsible for sending a `content` string, and the responsibility of creating the content is `eventprocessor`. Not sure `eventprocessor` should be responsible for that. That would imply passing the Receipt to `Send`. That change changes the meaning of Webhook from a simply HTTP content forwarder to a "Receipt Webhook" forwarder. 
- Maybe using `text/template` for producing the content, instead of `Sprinft`?